connector/keystone/keystone_test.go

```diff
@@ -0,0 +1,360 @@
+package keystone
+
+import (
+	"github.com/dexidp/dex/connector"
+	"testing"
+
+	"fmt"
+	"net/http"
+	"os"
+
+	"bytes"
+	"encoding/json"
+	"golang.org/x/net/context"
+	"io/ioutil"
+	"reflect"
+	"strings"
+)
+
+const (
+	adminUser   = "demo"
```

Comment:
❓ Would it make sense to pass these in, too, at least in so far as they're specific to https://hub.docker.com/r/openio/openstack-keystone?